src/population.py

```diff
@@ -0,0 +1,58 @@
+import random
+
+class Population():
+    def __init__(self, solutions, fitness):
+        self.solutions = solutions
+        self.fitness = fitness
+
+    def initial_population(self, population_size, test_size):
+        population = list()
+        for i in range(population_size):
+            test = list()
+            for j in range(test_size):
+                test.append(random.randint(-1000, 1000))
+            population.append(test)
+        return population
+
+    def mutate_1(self, lower, upper, prob):
+        mutated = list()
+        for test in self.solutions:
+            test_list = list(map(int, test.split()))
+            for i in range(len(test_list)):
+                if random.random() < prob:
+                    test_list[i] = random.uniform(lower, upper)
+            test_mutated = "".join(test_list)
+            mutated.append(test_mutated)
+        return mutated
+
+    def mutate_2(self, lower, upper, prob, rate):
+        mutated = list()
+        for test in self.solutions:
+            test_list = list(map(int, test.split()))
+            for i in range(len(test_list)):
+                if random.random() < prob:
+                    test_list[i] = random.uniform(lower, upper)
+            test_mutated = "".join(test_list)
+            mutated.append(test_mutated)
+        prob -= rate
+        return mutated
+
+    def crossover(self, parent_1, parent_2, rate):
```

Comment:
Ba pochemu ty zabil na to chto napisano v Class Diagram????

Reply:
Also you do not `rate` in here, it is already handled in `generator.py`

Reply:
ble oru